tensionmap.py

```diff
@@ -147,35 +154,60 @@ def tm_update(obj, context):
 
     # delete the temporary deformed mesh
     object_eval.to_mesh_clear()
-
+    
+    #create vertex color list for faster access
```

Comment:
Spaces between `#` and the comment.
https://www.python.org/dev/peps/pep-0008/#block-comments

I tried to follow the PEP8 style guide. However, as I now read the specs, I see they want capital letters at the beginning of comments.
I never used that in this file, so don't use it now. But the space would make it consistent with the rest of the file.

Reply:
Done.